sessionctx/variable/session.go

```diff
@@ -998,6 +997,25 @@ const (
 	SlowLogMemMax = "Mem_max"
 )
 
+type BackoffTimer struct {
+	waitScatterRegionFinish int
+}
+
+// GetWaitScatterRegionFinishBackoff gets the back off time of waitScatterRegionFinish.
+func (b *BackoffTimer) GetWaitScatterRegionFinishBackoff() int {
+	if b.waitScatterRegionFinish > 0 {
+		return b.waitScatterRegionFinish
+	}
+	return DefWaitScatterRegionFinishBackoff
+}
+
+// SetWaitScatterRegionFinishBackoff sets the back off time of waitScatterRegionFinish.
+func (b *BackoffTimer) SetWaitScatterRegionFinishBackoff(t int) {
```

Comment:
I think we needn't this function. `tidbOptPositiveInt32` is do it.

Reply:
done. Thanks.